### PR TITLE
Cancel Charge fees

### PR DESCRIPTION
**Pull request by owlee**
#263 

Changing :amount to :initial_amount, and changing every call to amount to be value()

### CODE REVIEW

File: app/models/financial_transaction.rb [hunk 1]
```diff
@@ -1,19 +1,22 @@
...
+  after_save :send_updated_invoice
+
   belongs_to :rental
   belongs_to :transactable, polymorphic: true
 
-  after_save :send_updated_invoice
-
   validates :adjustment, :rental_id, :amount, presence: true
-  validates :amount, numericality: { greater_than: 0 }
+  validates :amount, numericality: { greater_than: 0 }, presence: true
```
Comment (tomecho) [inline app/models/financial_transaction.rb]: why validate presence twice?

Comment (owlee) [inline app/models/financial_transaction.rb]: :man_cartwheeling: 💣 💥

File: app/models/rental.rb [hunk 2]
```diff
@@ -143,11 +146,11 @@ def self.to_json_reservations
   end
 
   def sum_charges
-    financial_transactions.where.not(transactable_type: Payment.name).sum(:amount)
+    financial_transactions.where.not(transactable_type: Payment.name).inject(0) {|total, transaction| total + transaction.balance}
```
Comment (owlee) [inline app/models/rental.rb]: Need to double check if works. Simpler way would be a map to balance

**Comment by owlee** [comment]
@tomecho Should I change :amount from incurred_incidental to :initial_amount also? It seems pretty harmless w/o the ':adjustments' that financial_transaction has.

**Comment by nawilliams** [comment]
Needed to change .amount to .initial_amount in the views too. 
when creating rental

I think I caught them all here: 0718612d1a62123302523b459c06f69bc677482f

**Pull request merged**